index.html

```diff
@@ -1,12 +1,216 @@
+            <div class="program-card-texts">
+              <h3 class="custom-red-2 text-14 big-text-20">CC Exhibition</h3>
+              <p class="custom-white text-10 big-text-12">
+                Appreciate various creations applying CC license of artists,
+                organized from Art Center Nabi
+              </p>
+            </div>
+          </div>
+          <div class="program-card">
+            <div class="icon-box">
+              <img src="./assets/img/forum-icon.png" />
+            </div>
+            <div class="program-card-texts">
+              <h3 class="custom-red-2 text-14 big-text-20">Forum</h3>
+              <p class="custom-white text-10 big-text-12">
+                Have the time to share your thoughts and opinions with experts
+                for each topic.
+              </p>
+            </div>
+          </div>
+          <div class="program-card">
+            <div class="icon-box">
+              <img src="./assets/img/workshop-icon.png" />
+            </div>
+            <div class="program-card-texts">
+              <h3 class="custom-red-2 text-14 big-text-20">Workshop</h3>
+              <p class="custom-white text-10 big-text-12">
+                Try creating your own work using open source license rather than
+                just watching at it.
+              </p>
+            </div>
+          </div>
+          <div class="program-card">
+            <div class="icon-box">
+              <img src="./assets/img/ignite-icon.png" />
+            </div>
+            <div class="program-card-texts">
+              <h3 class="custom-red-2 text-14 big-text-20">CC Ignite</h3>
+              <p class="custom-white text-10 big-text-12">
+                get opportunities to network with CC affiliates around the
+                world, also after the summit.
+              </p>
+            </div>
+          </div>
+        </div>
+        <button class="action-btn-2 flex-row custom-white text-16">
+          Join WD Global Summit 2023
+        </button>
+        <a class="btn-alt custom-white text-20-B" href="#">SEE THE WHOLE PROGRAM</a>
+      </div>
+    </section>
+    <!-- Speaker section-->
+    <section class="speakers-container flex-column">
+      <div class="speakers-content"></div>
+    </section>
+     <!-- partners section -->
+    <section class="partners-container index-partner">
+      <div class="partners-content">
+        <div class="partners-text flex-column">
+          <h3 class="text-20-C custom-white">Partner</h3>
+          <div class="title-underline"></div>
+        </div>
+        <div class="partners-logo flex-row">
+          <img class="logo-item" src="./assets/img/mozilla.png" />
+          <img class="logo-item" src="./assets/img/google.png" />
+          <img class="logo-item" src="./assets/img/naver.png" />
+          <img class="logo-item" src="./assets/img/daumkakao.png" />
+          <img class="logo-item" src="./assets/img/airbnb.png" />
+        </div>
```

Comment:
[REQUIRED] Please include the alt attribute and a value in all image tags that are not just meant for styling or design. But add an empty string as the value if the image is meant for styling and beautification.